static/app/views/replays/list/replaysList.tsx

```diff
@@ -87,6 +114,19 @@ function ReplaysListTable({
           ReplayColumns.countErrors,
           ReplayColumns.activity,
         ]}
+        emptyMessage={
+          allSelectedProjectsNeedUpdates && hasReplayClick ? (
+            <Fragment>
+              {t('The search query could not be made.')}
```

Comment:
This is an awkward sentence. Also we should drop the period, looks funny to be there when `NoRowRenderer` and the subtest don't have it.

An idea, idk:
```suggestion
              {t('Unindexed search field')}
```